app/components/lazy_wps_workflow_facet_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<turbo-frame id="wps-workflow-facet" src="<%= lazy_wps_workflow_facet_catalog_path(helpers.search_state.params_for_search) %>" target="_top">
```

Comment:
Making WPS workflow async because it is slow and hierarchical.

Reply:
but just this one for now as an experiment?   It's two brethren will be jealous

Reply:
Only this one shows on the home page.